src/applications/avs/components/YourTreatmentPlan.jsx

```diff
@@ -60,7 +73,14 @@ const YourTreatmentPlan = props => {
         orders={orders}
         type={ORDER_TYPES.OTHER}
       />
-      {/* TODO: add health reminders. */}
+      <ItemsBlock
+        heading="Your reminders"
```

Comment:
Hi-fis indicate this should read as "Health reminders". Has this changed?

Reply:
You're 100% right, thanks much!

Reply:
fixed, screenshot updated.